Our softhsm lib is in /usr/lib/softhsm.
TODO: We also need a pkcs11-tool (probably from Mastercard)

--- libp11-0.4.11/tests/rsa-common.sh.orig	2020-02-27 06:50:01.000000000 +0000
+++ libp11-0.4.11/tests/rsa-common.sh	2020-12-31 11:39:15.046963919 +0000
@@ -25,7 +25,7 @@
 
 # Set the module to be used
 for i in /usr/lib64/pkcs11 /usr/lib64/softhsm /usr/lib/x86_64-linux-gnu/softhsm \
-	/usr/local/lib/softhsm /opt/local/lib/softhsm /usr/lib/softhsm /usr/lib ;do
+	/usr/local/lib/softhsm /opt/local/lib/softhsm /usr/lib/softhsm /usr/lib /usr/lib/softhsm ;do
 	if test -f "$i/libsofthsm2.so"; then
 		MODULE="$i/libsofthsm2.so"
 		break
